IntegrationWithPharmacies/Controllers/ReportController.cs

```diff
@@ -32,7 +36,20 @@ public IActionResult Post(DateOfOrder date)
         {
             var sftpService = new SftpService(new NullLogger<SftpService>(), getConfig());
             var testFile = @"..\TextFile.txt";
-            System.IO.File.WriteAllText(testFile, getReportText(date));
+            StringBuilder stringBuilder2 = new StringBuilder();
+
+            List<RegistrationInPharmacy> registrations = RegistrationInPharmacyService.GetAll();
+            foreach (RegistrationInPharmacy registration in registrations)
+            {
+                stringBuilder2.Append(registration.ApiKey + ";");
+
+            }
+            String str = stringBuilder2.ToString();
+            str = str + "!";
+            str = str + "             Report about consumption of medicine\n\n\n";
+
+            str = str +getReportText(date);
+            System.IO.File.WriteAllText(testFile, str);

```

Comment:
Write this in form:
"System.IO.File.WriteAllText(testFile, stringBuilder2.ToString()+"!             Report about consumption of medicine\n\n\ngetReportText(date)");"